index.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module 'dronedeploy/datastore' {
+
+}
+
+
+
+declare module 'dronedeploy/functions-wrapper' {
+
+  import {Request, Response} from "express";
+
+  type Callback =  (err: Error | undefined, context: Context) => void;
+
+  interface Context {
+    token: string
+    originalToken: string
+  }
+
+  interface Cors {
+    headers?: string[],
+  }
+
+  interface Config {
+    authRequired: boolean,
+    mockToken?: string,
+    config?: Cors;
```

Comment:
I believe, what this is going to enforce is something that looks like this:

```
config: {
  headers: ...
}
```
which actually flattens out the `config` object some from what it's expecting:
```
"config": {
    "cors": {
      "headers": []
    }
  }
```

Reply:
Yes, that is wrong. I've updated it.